CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+
+
+## v0.7.0 (TBD)
+
+Starting a Changelog.
+
+### New features and changes
+
+* Dropped Python 2 support. Python >= 3.6 is now required.
+* Added new LicenseListVersion tag.
+* Added `pyspdxtools_convertor` and `pyspdxtools_parser` CLI scripts. See [the readme](README.md) for usage instructions.
+* Updated the tools to support SPDX versions up to 2.3 and to conform with the specification. Apart from many bugfixes 
+  and new properties, some of the more significant changes include:
+    * Support for multiple packages per document
+    * Support for multiple checksums for packages and files
+    * Support for files outside a package
+    * Removed example documents from production code. Added additional up-to-date examples to test files.
```

Comment:
This sounds like it's own point, not part of the sub list

Reply:
Yeah, fair enough. The reason it was there is that the new files are 2.2/2.3-specific. But it makes more sense as a point of its own